src/main/java/frc/robot/constants/ScoringConstants.java

```diff
@@ -1,6 +1,13 @@
 package frc.robot.constants;
 
 public class ScoringConstants {
+
+    public static enum ScoringStatesConstants {
+        Ground,
+        Amp,
+        Trap;
+    }
```

Comment:
i dont know if you need an extra enum since theres already one in scoring.java but wes probably knows

Reply:
I agree with griffin, we do not need this extra enum. Please use the one in the subsystem